packages/devtools_app/lib/src/service_manager.dart

```diff
@@ -650,6 +651,9 @@ class ServiceExtensionManager extends Disposer {
   ConnectedApp _connectedApp;
 
   Future<void> _handleIsolateEvent(Event event) async {
+    if (event.kind == EventKind.kIsolateExit) {
```

Comment:
will we receive this event for more isolates than the main isolate if a user is using more than one?

Reply:
Yes, definitely. I am not sure how to fix the actual issue...